contrib/localnet/orchestrator/start-zetae2e.sh

```diff
@@ -117,7 +117,7 @@ fund_eth_from_config '.additional_accounts.user_admin.evm_address' 10000 "admin
 fund_eth_from_config '.additional_accounts.user_migration.evm_address' 10000 "migration tester"
 
 # unlock precompile tests accounts
-fund_eth_from_config '.additional_accounts.user_precompile.evm_address' 10000 "precompile tester"
+fund_eth_from_config '.additional_accounts.user_precompile.evm_address' 10000 "precompiles tester"
```

Comment:
Is this needed as well to set the precompiles account under the auth module

Reply:
This is used to pay the precompiles e2e test used gas.